renovate.json5

```diff
@@ -169,6 +169,15 @@
       {
         packagePatterns: ['^@kbn/.*'],
         enabled: false,
+      },
+
+      // docs generator
+      {
+        packagePatterns: [
```

Comment:
Please use `packageNames` for absolute package names

Reply:
done